pkg/Microsoft.Private.PackageBaseline/stable.packages.props

```diff
@@ -1333,6 +1333,15 @@
     <StablePackage Include="System.IO.Compression">
       <Version>4.1.1</Version>
     </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.Platforms">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http">
+      <Version>4.1.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http.WinHttpHandler">
```

Comment:
Per discussion with @weshaggard, WinHttpHandler was not added to M.N.App in https://github.com/dotnet/core-setup/pull/586.

Do we still need to mark it stable?